src/main/java/Skeleton.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+
+/**
+ * Skeleton class for the Duke chatbox.
+ */
+
+public class Skeleton {
+
```

Comment:
Good naming convention! Overall all classes, easy to read and follow. Just missing packaging and javadocs! Great job